shared/components/TicketList/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import { formatDate } from '../../utilities/date';
+import classnames from 'classnames';
+
+export const TicketPrice = (ticket) => <strong>{ticket.available ? `£${ticket.price}` : 'SOLD OUT'}</strong>
+
+export function BuyTickets({ tickets }) {
+  const ticketsAvailable = tickets.some((ticket) => ticket.available);
```

Comment:
Tickets is listed as an optional prop, but this will crash if tickets is not an array.